setup/SeqCli.Setup/SeqCli.Setup.wixproj

```diff
@@ -49,7 +48,7 @@
   <Target Name="BeforeBuild">
     <PropertyGroup>
       <LinkerBaseInputPaths>..\..\src\SeqCli\bin\$(Configuration)\netcoreapp2.0\win-x64\publish</LinkerBaseInputPaths>
-      <DefineConstants>BasePath=..\..\src\SeqCli\bin\$(Configuration)\netcoreapp2.0\win-x64\publish</DefineConstants>
+      <DefineConstants>BasePath=..\..\src\SeqCli\bin\$(Configuration)\netcoreapp2.0\win-x64\publish;$(DefineConstants)</DefineConstants>
```

Comment:
This is the real fix here; the variable rename helps avoid confusion with built-in MSBuild and WiX variables.